client/my-sites/preview/main.jsx

```diff
@@ -66,23 +70,52 @@ class PreviewMain extends React.Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const { translate, isPreviewable, site } = this.props;
+
+		if ( ! site ) {
+			// todo: some loading state?
+			return <span></span>;
```

Comment:
hrm -- if I `return null;` here, & skip rehydration, I don't see an error. What are you seeing?